sites/main-site/src/content/events/2025/hackathon-march-2025/wellcome-genome-campus.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: 'Wellcome Genome Campus'
+subtitle: 'Local node of the nf-core hackathon at Wellcome Genome Campus, Hinxton, UK'
+shortTitle: '🇬🇧 Wellcome Genome Campus'
+type: 'hackathon'
+startDate: '2025-03-24'
+startTime: '09:00+01:00'
+endDate: '2025-03-26'
+endTime: '17:00+01:00'
+locations:
+    - name: Wellcome Genome Campus
+      address: |
+          Wellcome Genome Campus, CB10 1SA, UK
+      links:
+          - https://www.wellcomegenomecampus.org/
+      geoCoordinates: [52.079047, 0.187607]
+      country: United Kingdom
+      city: Hinxton
```

Comment:
```suggestion
      city: Cambridge
```

I think Cambridge is better, this is for the high level overview table, to differentiate from London for example.